intel_test/CMakeLists.txt

```diff
@@ -42,4 +42,12 @@ function(opencl_target KERNEL_NAME)
   add_custom_target(run_${KERNEL_NAME}_emulator COMMAND CL_CONTEXT_EMULATOR_DEVICE_INTELFPGA=1 ${CMAKE_CURRENT_BINARY_DIR}/Run${KERNEL_NAME}.exe emulator)
 endfunction()
 
-opencl_target("Jacobi2D")
\ No newline at end of file
+opencl_target("Jacobi2D")
+set(KERNEL_NAME "Jacobi2D")
+add_custom_target(run_${KERNEL_NAME}_hardware_oldapi_copy COMMAND ${CMAKE_CURRENT_BINARY_DIR}/Run${KERNEL_NAME}.exe hardware oldapi_copy)
+add_custom_target(run_${KERNEL_NAME}_emulator_oldapi_copy COMMAND CL_CONTEXT_EMULATOR_DEVICE_INTELFPGA=1 ${CMAKE_CURRENT_BINARY_DIR}/Run${KERNEL_NAME}.exe emulator oldapi_copy)
+add_custom_target(run_${KERNEL_NAME}_hardware_newapi_copy COMMAND ${CMAKE_CURRENT_BINARY_DIR}/Run${KERNEL_NAME}.exe hardware newapi_copy)
+add_custom_target(run_${KERNEL_NAME}_emulator_newapi_copy COMMAND CL_CONTEXT_EMULATOR_DEVICE_INTELFPGA=1 ${CMAKE_CURRENT_BINARY_DIR}/Run${KERNEL_NAME}.exe emulator newapi_copy)
+add_custom_target(run_${KERNEL_NAME}_hardware_newapi_notransfer COMMAND ${CMAKE_CURRENT_BINARY_DIR}/Run${KERNEL_NAME}.exe hardware newapi_notransfer)
+add_custom_target(run_${KERNEL_NAME}_emulator_newapi_notransfer COMMAND CL_CONTEXT_EMULATOR_DEVICE_INTELFPGA=1 ${CMAKE_CURRENT_BINARY_DIR}/Run${KERNEL_NAME}.exe emulator newapi_notransfer)
+unset(KERNEL_NAME)
```

Comment:
Why not just replace `${KERNEL_NAME}` everywhere with `Jacobi2D` directly, and get rid of this `set` and `unset`?

We should actually add this to CTest, but I can do that later.